.github/workflows/pytest.yml

```diff
@@ -0,0 +1,91 @@
+name: Python package
+
+on: [push]
+
+env:
+  FLASK_ENV: production
+  FLASK_DEBUG: 1
+  FLASK_APP: grc
+  LOG_LEVEL: DEBUG
+  TEST_URL: http://localhost:8080
+  SECRET_KEY: ef987sd9fwkjefosf9we0svs9q3evsd0
+  PERMANENT_SESSION_LIFETIME: 1200
+  SQLALCHEMY_DATABASE_URI: postgresql://postgres:password@localhost:5432/grc
+  SQLALCHEMY_TRACK_MODIFICATION: False
+  DEFAULT_ADMIN_USER: grc-service-account@cabinetoffice.gov.uk
+  NOTIFY_API: grctesting-36bdb0a3-86e3-423d-b1ce-26fae1ead417-869084fa-4e9b-4367-8803-b3115692d1cb
+  NOTIFY_SECURITY_CODE_EMAIL_TEMPLATE_ID: d93108b9-4a5b-4268-91ee-2bb59686e702
+  NOTIFY_UNFINISHED_APPLICATION_EMAIL_TEMPLATE_ID: 151fce32-1f66-4efd-a875-28026e8d8d70
+  NOTIFY_COMPLETED_APPLICATION_EMAIL_TEMPLATE_ID: d3a252f7-5580-4299-8889-01ac235e8de7
+  NOTIFY_ADMIN_LOGIN_TEMPLATE_ID: ddfa69ca-e89d-49d1-8311-b487732860ec
+  NOTIFY_ADMIN_FORGOT_PASSWORD_TEMPLATE_ID: 7e2ed682-d120-4937-9154-1966976e0144
+  NOTIFY_ADMIN_NEW_USER_TEMPLATE_ID: 0ff48a4c-601e-4cc1-b6c6-30bac012c259
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8"]
+
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_PORT: 5432
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
+      #- name: Lint with flake8
+      #  run: |
+      #    # stop the build if there are Python syntax errors or undefined names
+      #    flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      #    # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+      #    flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+      #    sudo -u postgres createdb grc
+      #    flask db stamp head
+      #    flask db init
```

Comment:
Do we still need these lines?
I'm keen to avoid commented-out code wherever possible